asm/battle_dome_cards.s

```diff
@@ -1277,5 +1277,4 @@ _0818D9A2:
 	bx r1
 	.pool
 	thumb_func_end sub_818D97C
-           
```

Comment:
was this really necessary? why remove the empty line?

Reply:
Probably automatic. Not sure why this is seen as a bad thing.